<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>265</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_inputs">
        <property name="title">
         <string>Input</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_SelectFolder">
           <property name="text">
            <string>Select Folder</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="input_folder">
           <property name="placeholderText">
            <string>Select a Folder to Convert</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_options">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="1" column="0">
              <widget class="QCheckBox" name="chk_resizeImg">
               <property name="toolTip">
                <string>Every Image bigger than the selected resolution will be downscaled to that Resolution</string>
               </property>
               <property name="text">
                <string>Resize Images</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cmb_resizeImg">
               <property name="toolTip">
                <string>Resize images to this Resolution.
Only images that are BIGGER Than the selected resolution will be resized</string>
               </property>
               <property name="statusTip">
                <string/>
               </property>
               <property name="currentIndex">
                <number>4</number>
               </property>
               <item>
                <property name="text">
                 <string>128</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>256</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>512</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1024</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2048</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4096</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="chk_restoreBackups">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="toolTip">
                <string>On: RestoreBackupFiles and optimize those files
Off: Use existing Var Files and optimize those</string>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Retore Backups</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoRepeat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <property name="formAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <item row="0" column="1">
              <widget class="QCheckBox" name="chk_recursive">
               <property name="toolTip">
                <string>Also Optimise Subfolders</string>
               </property>
               <property name="statusTip">
                <string/>
               </property>
               <property name="whatsThis">
                <string/>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Recursive (Subfolders)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_actions">
        <property name="title">
         <string>Actions</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="btn_Optimize">
           <property name="text">
            <string>Optimze !</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>395</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
